webapp/routes.go

```diff
@@ -40,6 +40,9 @@ func RegisterRoutes() {
 	// Dashboard for the interop effort, by year.
 	shared.AddRoute("/{name:(?:compat|interop-)}{year:[0-9]+}", "interop-dashboard", interopHandler)
 
+	// Default interop dashboard route that defaults to current year.
+	shared.AddRoute("/interop", "interop-dashboard", interopHandler)
```

Comment:
```suggestion
	shared.AddRoute("/{name:(?:compat|interop-)}{year:[0-9]+}", "interop-dashboard", interopHandler)
	shared.AddRoute("/interop", "interop-dashboard", interopHandler)
```
I think putting the two together is more clear, similar to testRunsHandler. Also merge the comment

Reply:
Actually a step further, given that /interop and /interop-202x are a part of each other, I think it is better to merge /interop to the regex `{name:(?:compat|interop-)}{year:[0-9]+}`. wdyt?

Reply:
I think that sounds like a good idea 🙂
I'll add it to the existing regex